src/lesson1/Cat.java

```diff
@@ -41,6 +48,7 @@ public Double getWeight()
     public String getStatus()
```

Comment:
Take out logic in separate class, entities must have only construnctors, getters, setter